undertow/src/main/java/org/wildfly/extension/undertow/handlers/FileHandler.java

```diff
@@ -80,7 +80,7 @@ public HttpHandler createHandler(final OperationContext context, ModelNode model
         String path = PATH.resolveModelAttribute(context, model).asString();
         boolean directoryListing = DIRECTORY_LISTING.resolveModelAttribute(context, model).asBoolean();
         UndertowLogger.ROOT_LOGGER.creatingFileHandler(path);
-        FileResourceManager resourceManager = new FileResourceManager(new File(path), 1024 * 1024);
+        FileResourceManager resourceManager = new FileResourceManager(new File(path), 1024 * 1024, true, new String[0]);
```

Comment:
this should be enabled by configuration option not by default.
similarly as it was done in https://issues.jboss.org/browse/WFLY-4254 for web applications